### ISSUE TITLE
[BUG] SenseVoice 2024 RKNN 模型输出异常，CPU ONNX 正常

### ISSUE THREAD

**Issue by hcc9902** [k2-fsa/sherpa-onnx#3600]
您好，我们在使用 SenseVoice 的 RKNN 模型时发现一个疑似兼容性问题，想咨询一下是否是已知问题。

一、环境信息
平台：RK3568
推理框架：sherpa-onnx（RKNN 后端）

二、测试模型
sherpa-onnx-rk3568-15-seconds-sense-voice-zh-en-ja-ko-yue-2024-07-17 （部分输出异常）
sherpa-onnx-rk3568-10-seconds-sense-voice-zh-en-ja-ko-yue-2024-07-17  （部分输出异常）
sherpa-onnx-rk3568-15-seconds-sense-voice-zh-en-ja-ko-yue-2025-09-09（该模型表现正常）

三、问题现象
在使用 2024 版本 RKNN 模型 时，ASR 输出存在异常情况，包括：
1. 输出文本出现中/英/韩等多语言混杂
2. 出现 <s>、</s> 等特殊 token 泄露
3. token 被拆分为无意义子词（如 drill / ler / ter）
4. 整体语义混乱，不符合正常识别结果

示例输出如下（包括正确输出和错误输出）：
<img width="2416" height="849" alt="Image" src="https://github.com/user-attachments/assets/7687a80b-9b45-46bb-9b93-5938fee3623c" />

四、对比情况
同一段音频测试结果如下：
使用 CPU ONNX（sense-voice 2024模型）：识别结果正常 
使用 RKNN（sense-voice 2025模型）：识别结果正常 
使用 RKNN（sense-voice 2024模型）：出现上述异常 

五、初步判断
从现象上看，问题仅出现在：RKNN + 2024 版本模型
推测可能与：
RKNN 推理结果（logits）分布异常 或 tokenizer / 解码过程在该模型版本下的兼容性有关

六、想咨询的问题
2024 版本的 SenseVoice RKNN 模型是否存在已知兼容性问题？
是否需要额外的配置或特定版本的 sherpa-onnx 才能正常使用？
